.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/Assistants/AzureAssistantClient.Protocol.cs

```diff
@@ -521,7 +521,7 @@ public override ClientResult DeleteThread(string threadId, RequestOptions option
     private new PipelineMessage CreateCreateAssistantRequest(BinaryContent content, RequestOptions options = null)
         => NewJsonPostBuilder(content, options).WithPath("assistants").Build();
 
-    private new PipelineMessage CreateGetAssistantsRequest(int? limit, string order, string after, string before, RequestOptions options)
+    private PipelineMessage CreateGetAssistantsRequest(int? limit, string order, string after, string before, RequestOptions options)
```

Comment:
Why did this change?

Reply:
The `new` keyword was resulting in a compiler error since there is no method to override as the updated create request method is called `CreateListAssistantsRequest` now. This error actually exists in the primary PR and looks like it was missed in the azure version of the sdk.